packages/docs/src/stories/components/dropdown.stories.ts

```diff
@@ -141,7 +141,7 @@ export const Rounded = {
     <sd-dropdown open rounded distance="4">
       <div class="slot slot--border slot--text p-4 w-[300px]">Default slot</div>
       <sd-button slot="trigger" variant="secondary">
-        <sd-icon name="system/more-functions" class="h-6 w-6"></sd-icon>
+        <sd-icon name="system/more-functions" class="h-6 w-6" label="Trigger rounded dropdown"></sd-icon>
```

Comment:
```suggestion
        <sd-icon name="system/more-functions" class="h-6 w-6" label="Open dropdown"></sd-icon>
```

Reply:
Maybe we should name this only `Rounded dropdown`, since "Open" should be renamed to "Closed" after it is already opened, and screenreader users could already identify if the dropdown is open or not, since it reads "expanded".

What do you think? @smfonseca 

```suggestion
        <sd-icon name="system/more-functions" class="h-6 w-6" label="Rounded dropdown"></sd-icon>
```

Reply:
Sounds good @paulovareiro29, go for it